staff/ana-sanjuan/mailchimp/styles.css

```diff
@@ -0,0 +1,249 @@
+@import url("https://fonts.googleapis.com/css?family=PT+Sans");
+.upperheader__logo, .upperheader__nav__ulist {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  margin: 0% 3%;
+}
+
+.upperheader__button, .lowerheader__button {
+  border-radius: 10px;
+  border: none;
+  height: 50px;
+  text-shadow: 1px 1px 5px black;
+  background-image: -webkit-gradient(linear, left bottom, left top, from(rgba(255, 255, 255, 0)), color-stop(50%, rgba(255, 255, 255, 0)), color-stop(51%, rgba(255, 255, 255, 0.1)));
+  background-image: linear-gradient(0deg, rgba(255, 255, 255, 0) 0%, rgba(255, 255, 255, 0) 50%, rgba(255, 255, 255, 0.1) 51%);
+  -webkit-box-shadow: 0px 1px 1px #3E3E37, 0px -1px 1px #3E3E37 open;
+          box-shadow: 0px 1px 1px #3E3E37, 0px -1px 1px #3E3E37 open;
+  cursor: pointer;
+  outline: 0;
+}
+
+body {
+  font-family: 'PT Sans';
+  background-color: black;
+}
+
+.container-all {
+  margin: 0 11%;
+}
+
+header {
+  background-color: #45BDD4;
+}
+
+.upperheader {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  margin: 1% 5% 1% 5%;
+}
+
+.upperheader__logo {
+  -webkit-box-flex: 30%;
+      -ms-flex: 30%;
+          flex: 30%;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  margin: 4% 0% 0% 2%;
+}
+
+.upperheader__button {
+  background-color: #3F3F37;
+  color: #DDDDDF;
+  width: 120px;
+  font-size: 12px;
+}
+
+.upperheader__nav-container {
+  -webkit-box-flex: 55%;
+      -ms-flex: 55%;
+          flex: 55%;
+  margin: 0% 2% 0% 0%;
+}
+
+.upperheader__p {
+  display: block;
+  text-align: right;
+  margin: 2% 7% 0% 2%;
+  color: white;
+}
+
+.upperheader__p a {
+  text-decoration: none;
+}
+
+.upperheader__nav__ulist {
+  background-color: #CFEEF4;
+  list-style-type: none;
+  -ms-flex-pack: distribute;
+      justify-content: space-around;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  border-right: #C3DEE3 1px solid;
+  margin-top: 1.1%;
+  border-radius: 10px;
+  border: none;
+  height: 50px;
+  text-shadow: 2px 2px 5px white;
+  -webkit-box-pack: end;
+      -ms-flex-pack: end;
+          justify-content: flex-end;
+}
+
+.upperheader__nav__ulist li {
+  border-right: 1px #adc8ca solid;
+  height: 50px;
+  line-height: 50px;
+  text-align: center;
```

Comment:
you have duplicated text-align: center and left.

Reply:
thanks!